### ISSUE TITLE
Resolving conflicts in notebooks

### ISSUE THREAD

**Issue by gogrean** [KIPAC/StatisticalMethods#62]
After making a few changes to my forked repository and then trying to bring it up-to-date with the upstream repository, I've gotten a couple of conflicts to deal with (mainly in the outputs). Resolving these conflicts turns out to be a bit of a mess. Right now I'm fixing conflicts with some brute force vim editing. Is there an easier way to deal with conflicts in notebooks?

For people using Python 2, there's a tool called NBDiff that looks promising.

Any magic trick to share for Python 3 users, other than committing only after stripping the output?

**Comment by abmantz** [comment]
Can you comment on how much trouble you get from
1. input differences
2. text output differences
3. graphical output differences?

If most or all of the conflicts are due to (3) or even (2+3), there's an obvious solution.

**Comment by gogrean** [comment]
I've only modified two files so far.

For XrayImage/Summarizing.ipynb, most of the conflicts (~10 cells) were related to the text output. These were the easiest to fix; I simply emptied the output cells. There were 2-3 conflicts related to differences in input. I don't recall conflicts caused by differences in graphical output.

For XrayImage/FirstLook.ipynb, all the conflicts (~5) were caused by differences in the text output.
